src/components/claiming/WalletEncryptionPassword.vue

```diff
@@ -15,21 +15,27 @@
       phrase is still your ultimate backup.
     </p>
     <div class="form-row password">
-      <PasswordInput
+      <p>Create a password</p>
+      <el-input
+        class="password"
         data-test="password-input"
-        label="Password"
+        placeholder="Please input password"
         v-model="password"
-        v-on:go-next="goNextInput"
-      />
+        show-password
+        @keydown.enter.native="goNextInput"
```

Comment:
Instead focus the next input on keydown enter should focus when pressing tab. I think you will find [tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) useful